core/services/pg/channels.go

```diff
@@ -2,7 +2,7 @@ package pg
 
 // Postgres channel to listen for new eth_txes
 const (
-	ChannelInsertOnEthTx     = "insert_on_eth_txes"
+	ChannelInsertOnTx        = "insert_on_eth_txes"
```

Comment:
Is there a plan to update the names in the db too?

Reply:
I refrained from making any DB changes since there will be a persistence layer rework anyway, but that should happen once the there's alignment on that direction